### PR TITLE
SUTD#2: western blot  band size fix

### LINKED ISSUE #112: SUTD Exercise 2: Adjustment to western blotting bands
from @laleman 

> Hi Anna. I just went through Exercise #2 again and we need to do a couple of adjustments to band intensities for western blotting for the anti-cyclin B and anti-Scc1 antibody conditions.  I have highlighted what it should look like under a new version of the excel coding file (Ver3) in yellow. For anti-cyclin B antibody condition, the change will make the blot look more realistic. For the anti-Scc1 condition, the change is to correct an error on my part.

### PR DESCRIPTION
**Pull request by annagav**
For testing: 
Perform the experiment, select western blot, select all samples, select gel type,
add marker, load gel, run gel and transfer)
Select primary antibody:
rabbit-antiScc1,
secondary:
goat anti-rabbit.

Fixes #112

### CODE REVIEW

**Comment by pdpinch** [comment]
Anna, could you give me some idea how to test this in the description?

This is for issue #112 right?

**Comment by pdpinch** [comment]
So this is what I got. Is it right?

screen shot 2015-03-18 at 11 12 32 am

**Comment by annagav** [comment]
This is the old version. There is one band missing.

**Comment by pdpinch** [comment]
Hmmm. Glad I asked. I probably didn't clear my account properly. Let me try again.

**Comment by pdpinch** [comment]
I created a new account, but I'm getting the same result. I'm starting to poke around the code to make sure that I picked up your changes.

**Comment by pdpinch** [comment]
Ok. I can see both changes now. 

I am slowly learning some biology.

**Comment by ald32** [comment]
:)
